### PR TITLE
adds unpublished icon to SingleEvent instances

### PR DESCRIPTION
**Pull request by michaelchadwick**
Fixes ilios/ilios#3473

Single Events need to show if they are unpublished just like when they're viewed on the calendar, so I added the missing icon. I also made sure to put missing mouseover text, and fixed up some tests.

### CODE REVIEW

File: packages/ilios-common/addon/components/single-event.hbs [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
-          {{#if (and this.sessionRouteExists this.canViewSession)}}
-            <LinkTo
-              @route="session"
-              @models={{array @event.course @event.session}}
-            >{{@event.name}}</LinkTo>
-          {{else}}
-            {{@event.name}}
-          {{/if}}
-        </em>
+        {{#if (not this.isPublished)}}
```
Comment (stopfstedt) [inline packages/ilios-common/addon/components/single-event.hbs:13]: should be checking `isPublished` on `@event`, not `this`. 

```suggestion
        {{#if (not @event.isPublished)}}
```

File: packages/ilios-common/addon/components/single-event.hbs [hunk 2]
```diff
@@ -1,26 +1,42 @@
...
+          <FaIcon
+            @icon="file-signature"
+            @title={{t "general.notPublished"}}
+            data-test-draft-icon
+          />
+        {{else if this.isScheduled}}
```
Comment (stopfstedt) [inline packages/ilios-common/addon/components/single-event.hbs:19]: should be checking `isScheduled` on `@event`, not `this`.

```suggestion
        {{else if @event.isScheduled}}
```

**Review by stopfstedt — CHANGES_REQUESTED** [review]
apart from the two issues mentioned in `single-event.hbs`, this looks good to me.

however, it's missing test coverage on these icons.

please add this appropriate tests.

for reference, i'd like to point you at existing page-object attributes and integration tests. please see https://github.com/ilios/frontend/blob/4718d3749813d2a3b96d063915db31febe7cd4f8/packages/test-app/tests/integration/components/weekly-calendar-event-test.js#L418-L486

**Comment by michaelchadwick** [comment]
Thanks for the review, @stopfstedt! I'll get back to this soon.

**Review by stopfstedt — APPROVED** [review]
LGTM

**Pull request merged**